lisa/tools/nvmecli.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from typing import List, Type, cast
+
+from lisa.executable import Tool
+from lisa.operating_system import Posix
+from lisa.tools import Git, Make
+
+
+class Nvmecli(Tool):
+    repo = "https://github.com/linux-nvme/nvme-cli"
+
+    @property
+    def dependencies(self) -> List[Type[Tool]]:
+        return [Git, Make]
```

Comment:
Because the dependencies is not necessary, when installing form package, let's call `install_packages` in `_install_from_src`, and not declare dependencies here.